safe_transaction_service/history/serializers.py

```diff
@@ -234,7 +235,14 @@ def validate(self, data):
 
     def save(self, **kwargs):
         safe_tx_hash = self.validated_data["contract_transaction_hash"]
-        origin = self.validated_data["origin"]
+        # Origin field on db is a JsonField
+        if self.validated_data["origin"] is not None:
```

Comment:
```suggestion
        if self.validated_data["origin"]:
```

This way we will be storing also empty strings as `{}`, so it's better for our database consistency

Reply:
Done [here](https://github.com/safe-global/safe-transaction-service/pull/1177/commits/a653677a60cc3b11930ffaf85fec5f50ee8a4dec)